web/src/settings_invites.ts

```diff
@@ -205,7 +205,6 @@ function do_resend_invite({$row, invite_id}: {$row: JQuery; invite_id: string}):
         },
         success() {
             $resend_button.text($t({defaultMessage: "Sent!"}));
-            $resend_button.removeClass("resend btn-warning").addClass("sea-green");
```

Comment:
This entire line is struck, because the `:disabled` styles override `resend btn-warning` whether those classes are present or not, and there is no discernible behavior attached to the absence of those classes, either.